rotkehlchen/db/utils.py

```diff
@@ -83,8 +83,9 @@ def form_query_to_filter_timestamps(
     """Formulates the query string and its bindings to filter for timestamps"""
     got_from_ts = from_ts is not None
     got_to_ts = to_ts is not None
+    query += ' '
```

Comment:
This alternative version does the same without adding this line (removing and adding spaces around, as well).
```python
    got_from_ts = from_ts is not None
    got_to_ts = to_ts is not None
    if got_from_ts or got_to_ts:
        query += ' WHERE ' if 'WHERE' not in query else 'AND '  # added leading and trailing whitespace

    filters = []
    bindings = []
    if got_from_ts:
        filters.append(f'{timestamp_attribute} >= ?')  # removed trailing whitespace
        bindings.append(from_ts)
    if got_to_ts:
        filters.append(f'{timestamp_attribute} <= ?')  # removed trailing whitespace
        bindings.append(to_ts)

    query += ' AND '.join(filters)  # added leading whitespace
    query += f' ORDER BY {timestamp_attribute} ASC;'  # added leading whitespace

    return query, tuple(bindings)
```

Reply:
lol yes I was about to tell you why you added another operation.